htdocs/core/class/html.formmail.class.php

```diff
@@ -755,7 +755,7 @@ public function get_form($addfileaction = 'addfile', $removefileaction = 'remove
 			if (!empty($this->withtoccuser) && is_array($this->withtoccuser) && getDolGlobalString('MAIN_MAIL_ENABLED_USER_DEST_SELECT')) {
 				$out .= '<tr><td>';
 				$out .= $langs->trans("MailToCCUsers");
-				$out .= '</td><td>';
+				$out .= '</td><>';
```

Comment:
This seems to be an error because a empty html tag like this isn't normal and we close a td on line 770 with no new td open